apps/prairielearn/src/lib/session-store.js

```diff
@@ -0,0 +1,92 @@
+// @ts-check
```

Comment:
This file could be converted to TypeScript! Since it's already being type-checked, that conversion should be straightforward. If you'd rather not do it yourself, let me know and I'll do it in a followup PR, though I'd recommend getting acquainted with TS because we'll mandate that all new code be TS in the near future.

Reply:
I'd rather get this merged and work on rewriting later. (Or you can hop on the branch and do it.) The long running PRs are problematic to leave open when the code base keeps changing and I'm tired of merging without landing.

I'm still waiting for guidance to the PL dev community about how to "get acquainted with TS". You've done a ton of work, but relatively little bringing us along with you.

Reply:
That's fair, we can land as-is. What kind of guidance are you looking for - links to the TS docs, examples of conversion PRs, a syntax 101, something else?

Reply:
I'm not sure, I just feel very ill prepared to do any dev in PrairieLearn these days and I'm not sure the path to fixing that.

Reply:
I know there's been a fair amount of rapid change and that can require some adjustment, but it will pay off in the long run - just ask @mwest1066 about his experience in PT before/after similar changes there! I'm happy to do whatever I can to get you and others better prepared, just let me know how I can help. Feel free to take this conversation to #pl-dev, this PR is probably not the best place to have it.